From 87155112b9311c764dca402907a636ef05a30d76 Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <r.stratiienko@gmail.com>
Date: Fri, 25 Oct 2024 12:51:52 +0300
Subject: [PATCH 15/19] common: Set BUILD_BROKEN_INCORRECT_PARTITION_IMAGES

Android-15 requires a make target for every file on the partition.
We're not ready for this change yet.

Signed-off-by: Roman Stratiienko <r.stratiienko@gmail.com>
---
 common/base/board.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/common/base/board.mk b/common/base/board.mk
index 9cc11e8..bd7c63e 100644
--- a/common/base/board.mk
+++ b/common/base/board.mk
@@ -178,3 +178,6 @@ BOARD_USES_GLODROID_MAKE_TARGETS := true
 BOARD_BUILD_GLODROID_KERNEL := true
 
 RUST_BIN_DIR := prebuilts/rust/linux-x86/1.68.0/bin
+
+# vendor_dlkm is not yet ready for files filtering
+BUILD_BROKEN_INCORRECT_PARTITION_IMAGES := true
-- 
2.43.0

